aries-site/src/pages/templates/filtering.mdx

```diff
@@ -1,124 +1,122 @@
+- In most cases when there are many available filters, they should be [presented in a layer](#filters-in-a-layer).
+- At times, it may be helpful to expose a [frequently used filter in the toolbar](#exposed-filter-in-toolbar) directly.
 
-### Results
+Different types of inputs may be better suited for filtering different types of data.
+
+When building with [Data and friends](/components/data), a smart default input for a given property will be determined based on the type of data and the number of options. For details on which/how the default input type is chosen, see [DataFilter documentation](/components/data#datafilter).
+
+While the default input types are intended as smart defaults for the 80% case, other input types can be used if it better suits the use case.
+
+## Results
 
 Results should live below the controls and display any records that meet filter criteria.
 Depending on the data types and use case, results may be represented by a
 [List](/templates/lists), [Cards](/components/card), or [DataTable](/components/datatable).
 
-### Pagination
+## Pagination
 
-If the number of records displayed per page is limited, pagination should be placed underneath the results.
+For large data sets, it can be helpful to paginate the data and display pagination controls underneath the results. For details on pagination behavior, see [Pagination documentation](/components/pagination).
 
-## When to use
+## Examples
 
 ### Filters in a layer
 
-When applying many filters at once, it is helpful to present them in a consolidated [Layer](/components/layer).
+When applying many filters at once, it is helpful to present them in a consolidated layer.
 
 <Example
   code={[
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js',
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringLists/mockData.js',
   ]}
-  showResponsiveControls={['laptop', 'mobile']}
+  docs="https://v2.grommet.io/data"
+  showResponsiveControls={['fullScreen']}
   template
 >
   <FilteringLists />
 </Example>
 
-### Filtering with selectable results
+### Exposed filter in toolbar
 
-The summary displayed beneath the search input should always reflect the user's actions. When filtering is combined with selection, the summary text should adjust for the following scenarios:
-
-- No filters, no selections &mdash; The summary should read "X items", where X is the total number of items.
-- Filters applied, no selections &mdash; The summary should read "X results of Y items".
-- Selections made &mdash; The summary should read "X of Y items selected", where Y is the total number of results that meet the filter criteria or the total number of items if no filtering has been applied.
+If it's expected that a specific filter will be most frequently used, it can be useful to expose it in the toolbar.
```

Comment:
Does this mean that at most one filter, the most frequently used, should be exposed? Or are there case where multiple filters could be exposed?

Reply:
Trying to leave the toolbar anatomy up to the designer based on the use case. So, not necessarily a "should" but more so a "can". This could be singular or multiple. I'll refine the wording slightly.

Reply:
Done.